packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -18,13 +18,27 @@ class PermissionCollection extends DocumentCollection {
     }
   }
 
-  async create({ _id, _type, ...attributes }) {
-    const resp = await this.stackClient.fetchJSON('POST', uri`/permissions`, {
-      data: {
-        type: 'io.cozy.permissions',
-        attributes
+  /**
+   * Create a new set of permissions
+   * It can also associates one or more codes to it, via the codes parameter
+   *
+   * @param {object} permission
+   * @param {string} permission.codes A comma separed list of values (defaulted to code)
+   *
+   * @see https://docs.cozy.io/en/cozy-stack/permissions/#post-permissions
+   *
+   */
+  async create({ _id, _type, codes = 'code', ...attributes }) {
+    const resp = await this.stackClient.fetchJSON(
+      'POST',
+      `/permissions?codes=${uri`${codes}`}`,
```

Comment:
I think we usually do 

```suggestion
      uri`/permissions?codes=${codes}`,
```

😁

Reply:
I don't want to espace the `?` since uri`` will return %3F  and the stack will not be able to decode it.

Reply:
I think it wouldn't encode it, it only encodes [the interpolated variables](https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/utils.js#L17). It's kinda cool tbh.